frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json

```diff
@@ -120,7 +121,7 @@
   "Provider {{provider}}": "Provider {{provider}}",
   "Create new BackingStore ": "Create new BackingStore ",
   "An error has occured while fetching backing stores": "An error has occured while fetching backing stores",
-  "Select a BackingStore": "Select a BackingStore",
+  "Select a backing store": "Select a backing store",
```

Comment:
I think previous was okay.

Reply:
for namespace store it is like this. For consistency i have kept the backing store in same way.